### DISCUSSION TITLE
Welcome, Caroline!

### DISCUSSION THREAD

**Discussion by Carol42** [ParanoidUser/yolo#828] [Subscribe]
Hi! I'd like to contribute to the project too! 😄

**Comment by ParanoidUser** [comment]
Ahoy, @Carol42!

Thank you for your interest in this project! We're very excited to have you as a collaborator. Do you have any specific things in mind that you would like to work on? We're constantly looking for new and innovative ideas to share with community.

Here are just a few of the areas where help is most needed:

- We're always on the lookout for more exciting questions & puzzles to share with the community;
- Create, improve, and maintain badge earning guides and materials;
- Increase user engagement and attract new users interested in the project;

**Reply by Carol42** [reply]
Can I submit trivia questions of any subject?

**Reply by ParanoidUser** [reply]
Absolutely! 🤓 As a rule of thumb, any general knowledge topic (such as science, art, music, geography, anagrams, etc.) that is familiar to a wide audience. There are no limits here, anything you find fun and interesting will work. 🙃

The invitation to join the collaborator group has been sent to you. Welcome aboard! 😊

**Reply by Carol42** [reply]
Great! I'm gonna think about some trivia questions and will post them soon.
Thank you for accepting me as a collaborator! :)
